pygmt/helpers/utils.py

```diff
@@ -209,9 +209,14 @@ def launch_external_viewer(fname):
 
     # Open the file with the default viewer.
     # Fall back to the browser if can't recognize the operating system.
-    if sys.platform.startswith("linux") and shutil.which("xdg-open"):
+    os_name = sys.platform
+    if (
+        os_name.startswith("linux")
+        or os_name.startswith("freebsd")
+        and shutil.which("xdg-open")
+    ):
```

Comment:
```suggestion
    if os_name.startswith(("linux", "freebsd")) and shutil.which("xdg-open"):
```
`str.startswith` accepts a tuple.